services/pull/check.go

```diff
@@ -88,6 +89,10 @@ func CheckPullMergable(ctx context.Context, doer *user_model.User, perm *models.
 		return ErrNotMergableState
 	}
 
+	if pr.IsChecking() {
```

Comment:
We still also need to check if it's in the checking queue.

Reply:
The status that's being checked in `IsChecking` is the status for those PR's that are currently in the queue.